opensrp-anc/src/main/java/org/smartregister/anc/library/util/ANCJsonFormUtils.java

```diff
@@ -773,6 +772,27 @@ public static Map<String, String> processSiteCharacteristics(String jsonString)
         }
     }
 
+    private static String getSelectedOptionValue(String value) {

```

Comment:
Can you add tests for the added functions?

Reply:
Added